### PR TITLE
Autocomplete component

### PR DESCRIPTION
**Pull request by jomaora**
**Detailed purpose of the PR**
This PR add a new component `autocomplete` as an atom and includes it as a possible input that can be used on `brand-form-group` molecule.

It also fixes a visual problem on with `select-multiple` molecule (background color and label not aligned)
Capture d’écran 2020-06-03 à 14 29 12

**Result and observation**

## Atom autocomplete
Capture d’écran 2020-06-03 à 14 26 08

## Visual fix of form having select and multiselect + autocomplete behavior
massive-battle-with-autocomplete

- **Extra lib ?**
      react-autosuggest(https://github.com/moroshko/react-autosuggest)

**Testing Strategy**

- Already covered by tests
- Manual testing
- Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/atom/autocomplete/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    modified = false,
+    error,
+    suggestions = [],
+    onChange = noop,
+    onFetch = noop,
+    onClear = noop,
+    onSuggestionSelected = noop
+  } = props;
+
+  const title = `${props.title}${required ? '*' : ''}`;
```
Comment (godu) [inline packages/@coorpacademy-components/src/atom/autocomplete/index.js:27]: `title` n'est pas dans les proptypes

File: packages/@coorpacademy-components/src/atom/autocomplete/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  );
+};
+
+Autocomplete.propTypes = {
+  placeholder: PropTypes.string,
+  value: PropTypes.string,
+  description: PropTypes.string,
+  required: PropTypes.bool,
+  modified: PropTypes.bool,
+  error: PropTypes.bool,
```
Comment (godu) [inline packages/@coorpacademy-components/src/atom/autocomplete/index.js:69]: `modified` et `error` ne semblent pas changer le theme du component.

**Comment by godu** [comment]
Je t'ai fait des PR pour améliorer storybook et ajouter le proptype manquant.
https://github.com/CoorpAcademy/components/pull/1738

File: packages/@coorpacademy-components/src/atom/autocomplete/index.js (hunk 2 above)
Comment (jomaora) [reply packages/@coorpacademy-components/src/atom/autocomplete/index.js:78]: @godu c'est bon, j'ai fait :)

**Pull request merged**